laos-indexer/docker-compose.yml

```diff
@@ -4,7 +4,7 @@ services:
   db:
     image: postgres:15
     environment:
-      POSTGRES_DB: squid
+      POSTGRES_DB: squid2
```

Comment:
Maybe a more explicit name, so later on I don't have to remember if this is laos or ownership